code/modules/mob/living/living.dm

```diff
@@ -353,6 +354,8 @@
 				mob_swap = TRUE
 			else if(mob_size > L.mob_size && a_intent == INTENT_HELP) //Larger mobs can shove aside smaller ones.
 				mob_swap = TRUE
+			else if((X.xeno_caste.caste_flags & CASTE_ALWAYS_PUSH_PAST) && isxeno(src))
+				mob_swap = TRUE
```

Comment:
I got bad news. Pushing a human as any xeno still does not work at all.

A little debugging showed me that `mob_swap = TRUE` is happening regardless of intent.

What if `L` and `X` as a result are not actually a xeno? I think that is causing the logic to break.

Reply:
-rep for not TESTING YOUR OWN SHIT MAGGOT
![monkey+keyboard](https://user-images.githubusercontent.com/64715958/95689573-8784f580-0bc6-11eb-8166-021cca0ce53d.gif)

Reply:
Ive been a little busy with like 4 other things

Reply:
I think we would all prefer delayed and tested code over speedy and untested. It is not like there is any pressure besides that which you put on yourself to get things pushed and PRed so quickly as to be error prone.